app/screens/DebugScreen.tsx

```diff
@@ -44,18 +46,34 @@ export const DebugScreen: FC<StackScreenProps<AppStackParamList, "Debug">> = ()
 
   return (
     <Screen
+      contentContainerStyle={$rootContainer}
       style={$root}
-      preset="scroll"
-      ScrollViewProps={{ showsVerticalScrollIndicator: false }}
+      preset="fixed"
       safeAreaEdges={["bottom"]}
     >
-      <Text preset="bold" tx="debugScreen.pushToken" style={$subtitle} />
-      <Text text={fcmToken} selectable />
-      <Button
-        shadowStyle={$resetStateButtonShadow}
-        tx="debugScreen.resetState"
-        onPress={() => clearState()}
-      />
+      <View>
+        <Text preset="bold" tx="debugScreen.pushToken" style={$subtitle} />
+        <Text text={fcmToken} selectable />
+        <Button
+          shadowStyle={$resetStateButtonShadow}
+          tx="debugScreen.resetState"
+          onPress={() => clearState()}
+        />
+      </View>
+      <View style={$footer}>
+        <Text preset="companionHeading" text={"Specs:"} />
+        {Updates.channel.length > 0 ?? (
```

Comment:
I wonder if the ?? operator is preventing this from displaying, maybe just a number is showing up somewhere for the length of the string? the others are ternary

Reply:
updating to `Updates.channel.length > 0 ? () : null` in next push